gimie/sources/gitlab.py

```diff
@@ -1,14 +1,287 @@
+from __future__ import annotations
+from dataclasses import dataclass
+import requests
+import os
+from datetime import datetime
+from typing import Any, Dict, List, Optional, Union
+from urllib.parse import urlparse
+
+from dotenv import load_dotenv
+from calamus import fields
+from calamus.schema import JsonLDSchema
 from rdflib import Graph
+
 from gimie.sources.abstract import Extractor
 
 
+from gimie.models import Organization, OrganizationSchema, Person, PersonSchema
+
+from gimie.graph.namespaces import SDO
+
+GL_API_rest = "https://gitlab.com/api/v4/"
+GL_API = "https://gitlab.com/api"
```

Comment:
How about having
`GL_API_REST` and `GL_API_GRAPHQL` as here (or just `GL_REST` and `GL_GRAPHQL`) ?